### PR TITLE
Answered the questions help! Declan Heritage

### PR DESCRIPTION
**Pull request by Declanhs**
# Your Details

- Your Name: Declan Heritage
- Your City: Manchester
- Your Slack Name: declanhs

# Homework Details

- Module: github
- Week: week 1/2?

### CODE REVIEW

**Comment by Anza-Azam** [comment]
Declan brilliant job!, just question 6 if we give identifier to h1 as id=h1 then we use #h1 otherwise we can use h1 for tags.

**Comment by Morteza-Khojasteh** [comment]
Good job

**Comment by Sharm-Dev** [comment]
Hi Declan, great work on these questions, I think my own comment would be changing question 6 to 
h1{
}
For the same explanation that Anza explains, but other then that really good 👍

File: homework.md [hunk 1]
```diff
@@ -92,19 +93,19 @@ A place for us to store and share our code.
 
 What does it mean to "Commit your changes"?
 
-Send you code to Github
+Send you code to Github and save it ready to be pulled into the code.
```
Comment (nax2uk) [inline homework.md:96]: Be careful. Commit your changes doesn't mean that you send your code to github. You can commit your changes without sending your code to github. If you want to send your code to github, this is done with the `git push` command after commiting your changes.
Committing your changes means:
1) adding the modified file to the staging area with a `git add` command.
2) creating a commit with that change `git commit`

**Review by nax2uk — COMMENTED** [review]
Great answers @Declanhs! 🎉
I have added some comments with links to help you understand and think about git on a deeper level. But otherwise, you've got this! 💪🏽

**Pull request closed without merging**